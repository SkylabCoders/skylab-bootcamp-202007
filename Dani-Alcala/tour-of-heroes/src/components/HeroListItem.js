import React from 'react';

function ListItem(props) {
	return <p>it works></p>;
}

export default ListItem;
