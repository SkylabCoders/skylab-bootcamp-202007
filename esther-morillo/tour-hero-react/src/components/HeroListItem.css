* {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

body {
    font-family: sans-serif;
    margin: 0 auto;
}

.title {
    color: #969696;
    margin-bottom: 1rem;
}


.hero__list {
    display: flex;
    justify-content: center;
    margin-bottom: .6rem;
    padding-left: .8rem;
    text-decoration: none;    
}


.hero__list span:nth-child(1) {
    display: inline-block;
    background-color: #557584;
    color: #fff;
    text-align: center;
    padding: .5rem;
    border-radius: .3rem 0  0 .3rem;
}

.hero__list a span:nth-child(2) {
    display: inline-block;
    background-color: #eee;
    color: #969696;
    padding: .5rem 0 .5rem .8rem;  
    border-radius: 0 .3rem .3rem 0;
    width: 8rem;
}
