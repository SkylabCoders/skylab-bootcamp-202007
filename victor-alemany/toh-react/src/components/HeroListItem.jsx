import React from 'react';

function HeroListItem(props) {
	return <p>HeroListItem works!</p>;
}

export default HeroListItem;