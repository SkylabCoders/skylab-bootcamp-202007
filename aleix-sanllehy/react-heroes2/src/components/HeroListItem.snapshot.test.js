import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import HeroListItem from './HeroListItem';

describe('HeroListItem snapshot', () => {
	const tree = renderer.create(
		<Router>
			<HeroListItem />
		</Router>
	);

	it('should match', () => {
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
