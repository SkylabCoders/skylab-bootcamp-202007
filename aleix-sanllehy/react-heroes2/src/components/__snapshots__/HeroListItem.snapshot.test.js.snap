// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroListItem snapshot should match 1`] = `
<p>
  HeroListItem works!
</p>
`;
